packages/terra-date-input/src/DateInput.jsx

```diff
@@ -246,9 +248,33 @@ class DateInput extends React.Component {
   }
 
   /**
- * Takes a key input from the month select, and processes it based on the value of the keycode.
- * @param {Object} event Event object generated from the event delegation.
- */
+   * Takes a key input and processes it based on if it is a hot key otherwise passes it to the apprioate input type key down handler.
```

Comment:
```suggestion
   * Takes a key input and processes it based on if it is a hot key otherwise passes it to the appropriate input type key down handler.
```